src/view/utils/MessageUtils.tsx

```diff
@@ -0,0 +1,9 @@
+interface vscode {
+    postMessage(message: any): void;

```

Comment:
Is it worth typifying this?

Reply:
this interface links to the vscode api which has any type. The only place where we call this specific interface is in sendMessage (with generic typing in the following comment you made)